lib/galaxy/tools/deps/__init__.py

```diff
@@ -110,7 +110,7 @@ def get_app_option(self, key, default=None):
 
     def dependency_shell_commands(self, requirements, **kwds):
         requirement_to_dependency = self.requirements_to_dependencies(requirements, **kwds)
-        return [dependency.shell_commands(requirement) for requirement, dependency in requirement_to_dependency.items()]
+        return OrderedSet(dependency.shell_commands(requirement) for requirement, dependency in requirement_to_dependency.items())
```

Comment:
@mvdbeek I'm refactoring the code to remove `requirement` from the `shell_commands()` call (see https://github.com/nsoranzo/galaxy/commit/13155ac2b56c8a207deb50b9e3e9682e501188fe ), so here we could push `OrderedSet` inside the list comprehension:
```python
return [dependency.shell_commands() for dependency in OrderedSet(requirement_to_dependency.values())]
```
What do you think?
I can rebase this on top of my changes, if you like.

Reply:
Great, yeah, that looks like a good change!